index.html

```diff
@@ -26,8 +26,125 @@ <h1 class="detail one">I'm Mariane<br>Glad to see<br>you</h1>
+        <ul class="tool1">
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>        
+        <div class="detail-style1">A daily selection of privately<br>reads:no account or<br>sign-up required</div>
+        <ul class="tool2">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javascript</a></li>
+        </ul>
+        <a href="#" class="detail-style2">See Project</a>
+      </div>
+    </div>
+    <div class="card">
+      <img src="image/Snapshoot-Portfolio-(2).png" alt="A photo of a website page">
+      <div class="card-content">
+        <div class="card-heading">Tonic</div>
+        <ul class="tool1">
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>        
+        <div class="detail-style1">A daily selection of privately<br>reads:no account or<br>sign-up required</div>
+        <ul class="tool2">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javascript</a></li>
+        </ul>
+        <a href="#" class="detail-style2">See Project</a>
+      </div>
+    </div>
+    <div class="card">
+      <img src="image/Snapshoot-Portfolio-(3).png" alt="A photo of a website page">
+      <div class="card-content">
+        <div class="card-heading">Tonic</div>
+        <ul class="tool1">
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>        
+        <div class="detail-style1">A daily selection of privately<br>reads:no account or<br>sign-up required</div>
+        <ul class="tool2">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javascript</a></li>
+        </ul>
+        <a href="#" class="detail-style2">See Project</a>
+      </div>
+    </div>
+    </div>
+
+    <div class="about-section">
+      <div class="myself">
+        <h1>About<br>Myself</h1>
+        <p class="about-detail">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href="" class="info-link">LET'S CONNECT</a>
+        <ul class="about-social">
+          <li><a href="#"><img src="image/Vector-(4).png" alt="Email icon"></a></li>
+          <li><a href="#"><img src="image/Linkedin-icon.png" alt="Social media"></a></li>
+          <li><a href="#"><img src="image/Vector-(1).png" alt="Email icon"></a></li>
+          <li><a href="#"><img src="image/Vector-(2).png" alt="Social media"></a></li>
+          <li><a href="#"><img src="image/Vector-(3).png" alt="Social media"></a></li>
+        </ul>
+        <a href="#" class="language">Get my resume</a>
+      </div>
+      <div class="language-detail">
+        <div class="lang icon">
+          <div class="word">Language</div>
```

Comment:
- [ ] Please change this main div (language-detail) container to a `ul` tag instead, and make all its direct child a `li`, please remember, all technologies should use `li` tag